### ISSUE TITLE
[mov,mp4,m4a,3gp,3g2,mj2 @ 0x2a191080] Could not find codec parameters for stream 1 (Audio: mp3 (mp4a / 0x6134706D), 44100 Hz, 2 channels, 127 kb/s): unspecified frame size

### ISSUE THREAD

**Issue by sapiosexual** [NVIDIA/DALI#3870]
Whenever I try to load video I receive the message 
```
[mov,mp4,m4a,3gp,3g2,mj2 @ 0x2a191080] Could not find codec parameters for stream 1 (Audio: mp3 (mp4a / 0x6134706D), 44100 Hz, 2 channels, 127 kb/s): unspecified frame size
Consider increasing the value for the 'analyzeduration' (0) and 'probesize' (5000000) options
```
As far as I understand there is some issue with ffmpeg codecs, but dali uses its own bundle of ffmpeg I have no idea how to access.

My env:
```
absl-py==1.0.0
aiohttp==3.8.1
aiosignal==1.2.0
async-timeout==4.0.2
attrs==21.4.0
av==9.2.0
cachetools==5.0.0
certifi==2021.10.8
charset-normalizer==2.0.12
cupy-cuda113==10.4.0
esn-recsys-lib==0.1.0
fastrlock==0.8
frozenlist==1.3.0
fsspec==2022.3.0
fvcore==0.1.5.post20220414
google-auth==2.6.6
google-auth-oauthlib==0.4.6
grpcio==1.45.0
idna==3.3
importlib-metadata==4.11.3
iopath==0.1.9
joblib==1.1.0
Markdown==3.3.6
multidict==6.0.2
numpy==1.22.3
nvidia-dali-cuda110 @ file://~/.cache/pypoetry/artifacts/a6/c6/c2/7b55900f593bacd801ae40169055b2f452a94d7cdab8c432120a7390a2/nvidia_dali_cuda110-1.13.0-4481327-py3-none-manylinux2014_x86_64.whl
oauthlib==3.2.0
packaging==21.3
pandas==1.3.5
parameterized==0.8.1
Pillow==9.1.0
portalocker==2.4.0
protobuf==3.20.1
pyasn1==0.4.8
pyasn1-modules==0.2.8
pyDeprecate==0.3.2
pyparsing==3.0.8
python-dateutil==2.8.2
pytorch-lightning==1.6.2
pytorchvideo==0.1.3
pytz==2022.1
PyYAML==6.0
requests==2.27.1
requests-oauthlib==1.3.1
rsa==4.8
scikit-learn==1.0.2
scipy==1.8.0
six==1.16.0
tabulate==0.8.9
tensorboard==2.9.0
tensorboard-data-server==0.6.1
tensorboard-plugin-wit==1.8.1
termcolor==1.1.0
threadpoolctl==3.1.0
torch @ file://~/.cache/pypoetry/artifacts/66/f3/bf/6e23024652b48b57c18ef85277fae948cf8b107332e01580e1203c4e87/torch-1.11.0%2Bcu113-cp38-cp38-linux_x86_64.whl
torchmetrics==0.8.1
torchvision @ file://~/.cache/pypoetry/artifacts/6f/61/da/883fc0493033e554c31fd80bd5e569c4ec95ea91b2c46d37b7ea143939/torchvision-0.12.0%2Bcu113-cp38-cp38-linux_x86_64.whl
tqdm==4.64.0
typing_extensions==4.2.0
urllib3==1.26.9
Werkzeug==2.1.2
yacs==0.1.8
yarl==1.7.2
zipp==3.8.0
```

Code to reproduce:
```
@pipeline_def
def train_video_pipe(filenames, side_size=224, crop_size=224, mean=np.array([[[[.45, .45, .45]]]]),
                     std=np.array([[[[.255, .255, .255]]]]), sequence_length=32,
                     initial_prefetch_size=4 * 32, sampling_rate=4):
    video, labels = fn.readers.video_resize(
        device="gpu", filenames=filenames, labels=[], sequence_length=sequence_length, random_shuffle=True,
        initial_fill=initial_prefetch_size, skip_vfr_check=True, size=(side_size, side_size),
        mode='not_smaller',
        dtype=types.FLOAT, normalized=True, stride=sampling_rate-1
    )

    # random crop
    video = fn.crop(video, crop=(crop_size, crop_size),
                    crop_pos_x=fn.random.uniform(range=(0.0, 1.0)),
                    crop_pos_y=fn.random.uniform(range=(0.0, 1.0)),
                    device='gpu')

    video = fn.normalize(video, mean=mean, stddev=std, device='gpu')

    video = fn.transpose(video, perm=[3, 0, 1, 2], device='gpu')

    slow, fast = fn.python_function(video, device='gpu', function=pack_pathway_dali, num_outputs=2)

    return slow, fast, labels

        train_pipeline = train_video_pipe(
            filenames=self.train_filenames,
            batch_size=self.batch_size,
            side_size=224, crop_size=224,
            mean=np.array([[[[.45, .45, .45]]]]),
            std=np.array([[[[.255, .255, .255]]]]),
            sequence_length=self.num_frames,
            initial_prefetch_size=self.batch_size * self.num_frames,
            sampling_rate=self.sampling_rate,
            num_threads=2,
            device_id=0,
            exec_async=False, exec_pipelined=False
        )

        train_pipeline.build()
```

Any idea what I can do?

**Comment by JanuszL** [comment]
Hi @sapiosexual,

Thank you for noticing this. This information tells that DALI doesn't support decoding of audio stream from the video files. You can disregard it. It doesn't affect the video reading performance.

**Comment by sapiosexual** [comment]
Hi @JanuszL 

Thank you for your fast response!

Is there a way to suppress this message? It is invoked for every video file in my dataset which is annoying?

Another question I would like to ask is there any way to read audio track alongside with video, but I guess "no" is an answer.

**Comment by JanuszL** [comment]
I don't think there is a way to do it now. We compile FFmpeg without audio codecs support.

> Another question I would like to ask is there any way to read audio track alongside with video, but I guess "no" is an answer.

I'm sorry but it is not supported now.

**Comment by sapiosexual** [comment]
Maybe it is possible to recompile it with audio codecs support and somehow replace it? Or maybe it is possible to force dali to use system installed ffmpeg?

**Comment by JanuszL** [comment]
We have the policy of `batteries includes` so we don't want to introduce any silent dependency on FFmpeg. We will discuss what it means for DALI if we enable audio codecs in FFmpeg.

**Comment by sapiosexual** [comment]
Okay, thank you for info.

**Comment by JanuszL** [comment]
I dig a bit deeper and we can turn this warning off - https://github.com/NVIDIA/DALI/pull/3871.
Please check the nightly/weekly build after it is merged to see if it solves your problem.

**Comment by JanuszL** [comment]
https://github.com/NVIDIA/DALI/pull/3871 was merged. Please check a nightly/weekly build that is built tomorrow or later to see if that solves your problem.
Also, it should be available in the 1.14 release that will be available by the end of May.

**Comment by sapiosexual** [comment]
I still have this output with `1.14.0dev.20220506` build.

**Comment by JanuszL** [comment]
@sapiosexual - the one you just tested was built in the very morning, while the fix was merged afternoon.

**Comment by sapiosexual** [comment]
I can confirm the issue was resolved with `1.14.0dev.20220508` build.
Thank you a lot!
